api-spec/protobuf/ark/v1/service.proto

```diff
@@ -336,4 +337,10 @@ message RedeemTransaction {
   string txid = 1;
   repeated Outpoint spent_vtxos = 2;
   repeated Vtxo spendable_vtxos = 3;
+}
+
+message MarketHour {
+  int64 first_market_hour = 1;  // Unix time representing the first market hour
+  int64 period = 2;             // Period between market hours
+  int64 round_lifetime = 3;     // Round lifetime in seconds
```

Comment:
Based on [issue](https://github.com/ark-network/ark/issues/297) desc i understood ASP sets timestamp of first marked hour and period, so that user invokes GetInfo and can always calculate when next round happens.
With this suggestion it seems to me that user will need frequently to call GetInfo. @altafan

Reply:
@altafan

Reply:
mmmm, this is not what i have in mind exactly. When a client calls get info he knows when the next market hour is intea of the very first one (likely in the past), from that moment, he uses the `period` to know when the next one will be.

Let's assume the server changes config, the clients will receive a notification to fetch get info and update their data about market hours. Added this to complete the reasoning but this last thing is not required to be implemented in this pr.

Reply:
@altafan in GetInfo should server calc next start_time (end_time) based on first_start_time, time.now() and round_interval?

Reply:
indeed, based on first start time, time.now() and period (not round interval, not needed)

Reply:
i was thinking to use round interval to calculate end_time @altafan, ok?

Reply:
Why not simplify to:
- next_market_hour_unixtime: for obvious reasons
- period_between_market_hours: to find out when will happen next market hours and how frequently they happen
- duration_of_market_hour: to know for how long will I need to be online to participate in this round

Reply:
isnt duration_of_market_hour same as round_interval? @bordalix